### PR TITLE
Send to external EOA

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/538?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#539** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/539?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#538** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/538?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#497** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/497?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/utils/useSearchResultHref.tsx [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+  const { search: query } = queryParams
   const path = usePathname()
   const [sendParams] = useSendScreenParams()
+  if (!profile)
+    switch (true) {
+      case !query || !isAddress(query):
+        return ''
+      case path === '/activity':
+        return `https://etherscan.io/address/${query}`
+      case path === '/send':
```
Comment (0xBigBoss) [inline packages/app/utils/useSearchResultHref.tsx:17]:
```suggestion
import { baseMainnet } from '@my/wagmi/chains'
//...
        return `https://${baseMainnet.blockExplorers.default.url}/address/${query}`
```

File: packages/app/utils/useSearchResultHref.tsx [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
 import type { SearchResultCommonType } from 'app/components/SearchBar'
+import { isAddress } from 'viem'
 
-export const useSearchResultHref = (profile: SearchResultCommonType) => {
+export const useSearchResultHref = (profile?: SearchResultCommonType) => {
+  const [queryParams] = useRootScreenParams()
+  const { search: query } = queryParams
   const path = usePathname()
   const [sendParams] = useSendScreenParams()
+  if (!profile)
```
Comment (0xBigBoss) [inline packages/app/utils/useSearchResultHref.tsx:11]: add braces {} pretty please

File: packages/app/features/send/confirm/screen.tsx [hunk 3]
```diff
@@ -198,7 +206,9 @@ export function SendConfirm({ profile }: { profile: ProfileProp }) {
     }
   }, [sentTxHash, transfers, router, sendToken, tokenActivityError, dataFirstFetch, dataUpdatedAt])
 
-  if (balanceIsLoading || nonceIsLoading) return <Spinner size="large" color={'$color'} />
+  if (balanceIsLoading || nonceIsLoading || isProfileLoading)
+    return <Spinner size="large" color={'$color'} />
+  if (profileError) throw new Error(profileError.message)
```
Comment (0xBigBoss) [inline packages/app/features/send/confirm/screen.tsx:211]: would rather show this in the UI instead of crashing the app

Comment (youngkidwarrior) [reply packages/app/features/send/confirm/screen.tsx:211]: This actually gets handled in the `<SendRecipient>` component anyway. no need to handle it here

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jul 9, 4:19 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/538).

**Pull request merged**